### DISCUSSION TITLE
Looking for better ideas to workaround garbo attempting mushroom garden YR (aftercore WoL)

### DISCUSSION THREAD

**Discussion by eric-the-redbeard** [loathers/garbage-collector#1907] [Q&A]
I don't have mushroom garden, but garbo gets stuck in a loop trying to adventure there and YR some target. I am in WoL (post prism), though, so that is probably part (or all) of it. It did run for a few days just fine in WoL aftercore, just not lately.

My current workaround attempt is to just YR almost anything else, so that the cooldown effect is up, and that seems to be working. So just have to interrupt every 100 turns. Not bad. I don't want to enter it as an issue since it is a weird case. Just looking for better advice if any, for a workaround.

Here's the snippet of what it's doing:

Visit to Campground: Your Mushroom Garden in progress...

[617] Your Mushroom Garden
Encounter:
Nothing more to do here.

Executing Wanderers/Spit Acid
Wandering at Your Mushroom Garden for expected value 47320.00 (Yellow Ray Your Mushroom Garden)

**Comment by Tokoeka** [comment]
Do you happen to know what sort of wandering monster if was trying to send to the mushroom garden?

**Comment by eric-the-redbeard** [comment]
https://kol.coldfront.net/thekolwiki/index.php/Piranha_plant

And of course today it didn't have the behavior. It appears to be the only monster in that zone.

Must've been trying to do it based on mall value for its drops.
